apps/karma.conf.js

```diff
@@ -47,7 +49,8 @@ module.exports = function (config) {
       plugins: [
         new webpack.ProvidePlugin({React: 'react'}),
         new webpack.DefinePlugin({
-          IN_UNIT_TEST: true,
+          IN_UNIT_TEST: JSON.stringify(true),
```

Comment:
Also, for the false version in the gruntfile, i see that we don't convert to a string. Is there a reason we do here?